.circleci/config.yml

```diff
@@ -70,6 +70,44 @@ jobs: # a collection of steps
           # Upload test results for display in Test Summary: https://circleci.com/docs/2.0/collect-test-data/
           path: target/surefire-reports
 
+      - persist_to_workspace:
+          root: target/components/packages
+          paths:
+            - scalyr-kafka-connect-scalyr-sink-<< pipeline.parameters.scalyr_sink_version >>.zip
       - store_artifacts: # store the jar as an artifact
           # Upload test summary for display in Artifacts: https://circleci.com/docs/2.0/artifacts/
           path: target/components/packages/scalyr-kafka-connect-scalyr-sink-<< pipeline.parameters.scalyr_sink_version >>.zip
+          destination: kafka_connect_scalyr_plugin
+
+  publish-github-release:
+    docker:
+      - image: circleci/golang:1.12
+    steps:
+      - attach_workspace:
+          at: ./artifacts
+      - run:
+          name: "Publish Release on GitHub"
+          command: |
+            go get github.com/tcnksm/ghr
+            VERSION=<< pipeline.parameters.scalyr_sink_version >>
+            echo Releasing version $VERSION
+            ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION} ./artifacts
```

Comment:
I know Circle CI tries to mask secret values in the output, but it's probably still safer to specify as environment variable.

It looks like the environment variable for ``ghr`` is called ``GITHUB_TOKEN`` as well, so it should work out of the box as long as that Circle CI secret is set.

Having said there, we should add a comment to this step and say that ``GITHUB_TOKEN`` secret needs to be defined.

Reply:
I also assume we will use one of the bot users for this github oauth access token?

My main issue with oauth access tokens is that we can't really only grant "manage releases" access for a single repo - it's almost all or nothing which is really not great and means we likely should create new dedicated Github bot user and only give him admin / manage releases access to this repo.

Reply:
And as mentioned on Slack - since we are passing secret to this go app, it would be safer to pin that go get to a specific repo revision (aka ``go get github.com/foo@[gitref]``).

Reply:
I wasn't aware we have bot users.  Following the directions here https://circleci.com/blog/tips-for-optimizing-docker-builds/ , I created a Github personal access token with my account using these directions: https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token

Reply:
Changed to use `GITHUB_TOKEN` to use the `scalyr-cloud-tech-automations` user.  

I looked into pinning the version for `go get`, but it looks like `go get` doesn't support that:
```
go get github.com/tcnksm/ghr@d43a5d2
go: cannot use path@version syntax in GOPATH mode

go get github.com/tcnksm/ghr/releases/tag/v0.13.0
package github.com/tcnksm/ghr/releases/tag/v0.13.0: cannot find package "github.com/tcnksm/ghr/releases/tag/v0.13.0" in any of:
	/usr/local/go/src/github.com/tcnksm/ghr/releases/tag/v0.13.0 (from $GOROOT)
	/Users/oliver/go/src/github.com/tcnksm/ghr/releases/tag/v0.13.0 (from $GOPATH)
```

Search also says this is not supported and requires 3rd party tools to do this:
https://stackoverflow.com/questions/30188499/how-to-do-go-get-on-a-specific-tag-of-a-github-repository
https://github.com/koofr/go-pin

I'm not sure it's worth using a 3rd party tool to try to pin the version.

Ready for another look.

Reply:
Yeah, I think it's fine and less of an issue now that we use more scoped token.